### ISSUE TITLE
Incorrect syntax for powershell.

### ISSUE THREAD

**Issue by Layonas** [lalilaloe/wttab#12]
Just started using this library and I am sorry if this is incorrect but tried running the code from package.json `"wttab -t 'Run API' npm run run:api"` and when windows terminal opens it throws an error:
![image](https://user-images.githubusercontent.com/60541384/166956205-bf84809c-3206-4a3d-b5da-88a467e8e116.png)
When added debug statement `"wttab -t 'Run API' npm run run:api --debug"` i see that it tried running a command:
`"wt.exe" -w 0  --title "'Run" -p "Windows PowerShell" -d "dir/path" API',npm,run,run:api`
While testing the wt command I found out that the correct syntax should have been:
`wt.exe -w 0  --title "Run API" -p "Windows PowerShell" -d "dir/path" powershell npm run run:api`
This runs the code nicely.

Is it just my mistake or something is actually wrong?

**Comment by lalilaloe** [comment]
I will look into this, and add some more testing.

**Comment by malhar016** [comment]
Did this ever work after the fix?
I am facing exactly same issue.

Kindly help.

**Comment by malhar016** [comment]
OMG it worked after spending few hours on this 😄 

Solution: use double quotes-" only and escape them.
"wttab -t /"Run API/" /"npm run run:api/""

**Comment by lalilaloe** [comment]
@malhar016 Yes if you use it in npm package JSON you need to escape the quotes. I'm working towards a new release with test cases, so if you encounter any other issues let me know
